frontend/app/views/fragments/tier/packagePromo.scala.html

```diff
@@ -51,19 +54,35 @@
                 }
             </div>
         }
-        <div class="package-promo__actions">
-            <div class="package-promo__actions__pricing">
-               @plans match {
-                   case FreePlan(_) => { @fragments.pricing.freePriceInfo() }
-                   case PaidPlans(p) => { @fragments.pricing.paidPriceInfo(p, cg.currency) }
-               }
+
+            <div class="package-promo__actions">
+                <div class="package-promo__actions__pricing">
+                @plans match {
+                    case _ if customPrice.isDefined => {
+                        @customPrice.get
+                    }
+                    case FreePlan(_) => {
+                        @fragments.pricing.freePriceInfo()
+                    }
+                    case PaidPlans(p) => {
+                        @fragments.pricing.paidPriceInfo(p, cg.currency)
+                    }
+                }
+                </div>
+                <a class="action qa-package-@plans.tier.slug u-fullwidth  @if(!secondaryButton.isDefined){u-no-bottom-margin}" href="@promoButton.to" @promoButton.attributes.html>
+                    @promoButton.text
+                </a>
+                @for(button <- secondaryButton){
+                    <br/>
```

Comment:
avoid using `<br />` tags just to space sections. https://developer.mozilla.org/en/docs/Web/HTML/Element/br